apps/mobile/src/components/NftPreview/NftPreviewContextMenuPopup.tsx

```diff
@@ -0,0 +1,192 @@
+      if (event.nativeEvent.actionKey === 'view-details') {
+        navigation.navigate('NftDetail', { tokenId: token.dbid });
+      } else if (event.nativeEvent.actionKey === 'share') {
+        shareToken(token);
+      }
+    },
+    [navigation, token]
+  );
+
+  return (
+    <ContextMenuView
+      // If we don't have a tokenUrl, we should bail
+      isContextMenuEnabled={Boolean(tokenUrl)}
+      onPressMenuItem={handleMenuItemPress}
+      menuConfig={{
+        menuTitle: '',
+        menuItems: [
+          {
+            actionKey: 'view-details',
+            actionTitle: 'View Details',
+          },
+          {
+            actionKey: 'view-gallery',
+            actionTitle: 'View Gallery',
+          },
+          {
+            actionKey: 'share',
+            actionTitle: 'Share',
+          },
+        ],
+      }}
+      previewConfig={{
+        previewType: 'CUSTOM',
+        previewSize: 'INHERIT',
+      }}
+      renderPreview={() => {
+        if (!tokenUrl) {
+          throw new Error("NftPreviewContextMenu shown without a tokenUrl. This shouldn't happen.");
+        }
+
+        const MAX_WIDTH = windowDimensions.width - 40;
+        const MAX_HEIGHT = 400;
+
+        const finalDimensions = fitDimensionsToContainer({
+          container: { height: MAX_HEIGHT, width: MAX_WIDTH },
+          source: imageDimensions ?? { width: MAX_WIDTH, height: MAX_WIDTH },
+        });
+
+        return (
+          <View className="bg-white">
+            <View className="self-center" style={finalDimensions}>
+              <NftPreviewAsset
+                priority="high"
+                tokenUrl={tokenUrl}
+                resizeMode={ResizeMode.CONTAIN}
+                onLoad={handlePopupAssetLoad}
+              />
+
+              {popupAssetLoaded ? null : (
+                <View className="absolute inset-0">
+                  <GallerySkeleton>
+                    <SkeletonPlaceholder.Item width="100%" height="100%" />
+                  </GallerySkeleton>
+                </View>
+              )}
+            </View>
+            <View className="flex flex-col space-y-2 p-4">
+              <Typography
+                className="text-2xl"
+                font={{ family: 'GTAlpina', weight: 'Light', italic: true }}
+              >
+                {token.name}
+              </Typography>
+              <View className="flex flex-row space-x-2">
+                {token.owner?.username && (
+                  <View className="flex-1 flex-col">
+                    <Typography
+                      className="text-xs"
+                      font={{ family: 'ABCDiatype', weight: 'Medium' }}
+                    >
+                      CREATED BY
+                    </Typography>
+
+                    <Typography
+                      className="text-shadow text-sm"
+                      font={{ family: 'ABCDiatype', weight: 'Regular' }}
+                    >
+                      {token.owner.username}
```

Comment:
im realizing CREATED BY doesn't fully make sense to have yet before we launch Creator Support. The idea would be to show the artist name here.


And we probably don't need to show the owner username here either because we already know the owner from the context of the feed event or gallery page, and it's also available on the detail page.

So, I'm thinking maybe we can simplify the metadata display here to only show the title and community name.

This way we can keep the preview super clean and intentional until we have a better reason to display additional fields, like when we have Creator Support and can display the artist name.
![Screenshot 2023-04-06 at 16 13 17](https://user-images.githubusercontent.com/80802871/230302233-ee77acd1-e820-4ee5-bc4c-09383e376d96.png)

Reply:
As a bonus this should also help resolve some other edge cases I'm seeing where the community name is long and doesn't fit.
the updated design above will create more space to display the community name. We can cap at 1 line and truncate with ellipses for anything longer than that.

![Screenshot 2023-04-06 at 15 48 27](https://user-images.githubusercontent.com/80802871/230302416-2dc5b6ea-8f33-4446-af5c-097456290cda.png)